recipes/stereo-pipeline/build.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+# Architecture-dependent flags
+if [ "$(uname)" = "Darwin" ]; then
+    LIB_FLAG='-dynamiclib'
+    deployFlag="-DCMAKE_OSX_DEPLOYMENT_TARGET=10.13"
```

Comment:
This is unneeded.

Reply:
I took out the CMAKE_OSX_DEPLOYMENT_TARGET. The -dynamiclib seems to be needed.